org.testeditor.web.dropwizard.xtext/src/main/java/org/testeditor/web/dropwizard/xtext/XtextConfiguration.xtend

```diff
@@ -10,6 +10,9 @@ public class XtextConfiguration extends DropwizardApplicationConfiguration {
 
 	@NotEmpty @JsonProperty
 	String localRepoFileRoot = 'repo'
+	
+	@JsonProperty
+	String[] indexSearchPaths = #['/src/main/java', '/src/test/java', '/build/classes/java/main']
```

Comment:
does it make sense to separate source paths from binary paths? do we run in problems, if the path to binaries holds compiled resources that are present (as source) on the source path too (eclipse sometimes reports something like 'java class xyz already exsist')?

Reply:
That's a valid point, I'll try to draw up tests for that, and try to address this issue somehow.
Since this requires actual change detection taking place, I would prefer to introduce that with the next round of changes, if you don't mind.